style.css

```diff
@@ -0,0 +1,138 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  margin-top: 10px;
+}
+
+body {
+  position: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  background: #fff;
+}
+
+.toolbar {
+  left: 0;
+  right: 0;
+  top: 40px;
+  bottom: 0;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.logo {
+  width: 100px;
+  height: 40px;
+  left: 12px;
+  padding-bottom: 10px;
+  font-family: "Poppins", sans-serif;
+  margin-left: 100px;
+}
+
+.icons {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  border-radius: 8px;
+  margin-right: 90px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  position: absolute;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+  background-image: url(Images/header.png);
+}
+
+.header-shapes {
+  position: absolute;
+  width: 462px;
+  height: 587.15px;
+  left: -34px;
+  top: 104px;
+  display: flex;
+  align-items: center;
+}
+
+.text1 {
+  width: 327px;
+  height: 150px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.text2 {
+  width: 327px;
+  height: 146px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.letsgo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 117px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
+}
+
+.socialmediaicons {
+  display: flex;
+  gap: 20px;
+  margin-left: 0;
+  list-style-type: none;
+}
```

Comment:
- [OPTIONAL] I suggest to use `align-items:center` to make sure the icons are at the same height:

 - Your design:
![youSocial](https://user-images.githubusercontent.com/97834160/206290986-f9ce1f0f-8b6c-421a-9c52-28ca48e45a10.png)


 - Figma design:
![figSocial](https://user-images.githubusercontent.com/97834160/206291150-3ceed558-3d98-440e-8ffd-138e38ee9ec7.png)